package.json

```diff
@@ -29,5 +36,8 @@
   "license": "MIT",
   "bugs": {
     "url": "https://github.com/lachlanjc/calltoaction/issues"
+  },
+  "devDependencies": {
+    "sync-exec": "^0.6.2"
```

Comment:
could you please move `devDependencies` up so that it's right below `dependencies`?